smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/DefaultHTTPProtocolCustomizations.kt

```diff
@@ -35,13 +34,14 @@ abstract class DefaultHTTPProtocolCustomizations : HTTPProtocolCustomizable {
 
     override fun renderInternals(ctx: ProtocolGenerator.GenerationContext) {
         AuthSchemeResolverGenerator().render(ctx)
+        EndpointParamsGenerator(ctx).render()
         EndpointResolverGenerator(
             partitionDefinition = ClientRuntimeTypes.Core.PartitionDefinition,
-            dependency = SwiftDependency.CLIENT_RUNTIME,
-            endpointResolverMiddleware = { w, i, o, oe -> EndpointResolverMiddleware(w, i, o, oe) }
         ).render(ctx)
     }
 
+    override val endpointMiddlewareSymbol: Symbol = ClientRuntimeTypes.Core.EndpointResolverMiddleware
```

Comment:
The endpoint middleware symbol is configurable so that AWS & Smithy can have their own types.